sh/tests/entrypoint.sh

```diff
@@ -6,7 +6,9 @@ cp -r /app .
 cp -a student app
 cd app
 
-chmod +x "./student/${EXERCISE}.sh"
+if test -f "./student/${EXERCISE}.sh"; then
+	chmod +x "./student/${EXERCISE}.sh"
+fi
```

Comment:
Would this be better?

```bash
find /student -type f -name "*.sh" -exec chmod +x {} \;
```

Reply:
Thanks a lot for the suggestion, actually I initially tough about this approach, but I'm not sure we want all files to be executable (my fear is that some exercises are indeed about `chmod` and we may have random failures by forcing a specific state on all files in there)

Reply:
Ah, ok then it should be good!